### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by jinfagang** [jinfagang/alfred#1]
```
 alfred data voc_view -i JPEGImages -l Annotations
Alfred - Valet of Artificial Intelligence.
Author: Lucas Jin
At    : 2018.11.11
Loc   : Shenzhen, China
Star  : http://github.com/jinfagang/alfred
Ver.  : 2.5.15

=> Module: scrap
=> Action: image
 parse args error, type -h to see help. msg: 'query'

```

**Comment by paohaijiao** [comment]
how can i run pip3 install alfred-py in win10?
iget the follow error
:\soft\anaconda\envs\keras\Scripts>pip install alfred-py
Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
Collecting alfred-py
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/67/14/e7a84eebdd8c7b117ba208ceab306fa521f0fc57b25147f8141d8343e725/alfred-py-2.6.7.tar.gz (66kB)
     |████████████████████████████████| 71kB 1.2MB/s
    ERROR: Command errored out with exit status 1:
     command: 'D:\soft\anaconda\envs\keras\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\pip-install-lrha5xw2\\alfred-py\\setup.py'"'"'; __file__='"'"'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\pip-install-lrha5xw2\\alfred-py\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\ADMINI~1\AppData\Local\Temp\pip-install-lrha5xw2\alfred-py\pip-egg-info'
         cwd: C:\Users\ADMINI~1\AppData\Local\Temp\pip-install-lrha5xw2\alfred-py\
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\ADMINI~1\AppData\Local\Temp\pip-install-lrha5xw2\alfred-py\setup.py", line 26, in <module>
        with open("readme.md", "r") as fh:
    FileNotFoundError: [Errno 2] No such file or directory: 'readme.md'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

**Comment by jinfagang** [comment]
@paohaijiao  Thanks for point out this. We should have fixed it in 2.6.8 release. just `pip install -U alfred-py` to upgrade it.
